CMakeLists.txt

```diff
@@ -22,7 +22,7 @@ set(CMAKE_CXX_STANDARD 17)
 
 
 # Select the test project to build
-set(selected_test_project starter_example)
+set(selected_test_project access_point)
```

Comment:
Please keep the starter project as the default.

Reply:
OK

Reply:
This is still showing access_point as the selected project.